packages/ckeditor5-emoji/src/emojirepository.ts

```diff
@@ -219,24 +220,97 @@ export default class EmojiRepository extends Plugin {
 		return this._repositoryPromise;
 	}
 
+	/**
+	 * Returns the URL from which the emoji repository is downloaded. If the URL is not provided
+	 * in the configuration, the default URL is used with the version from the configuration.
+	 *
+	 * If both the URL and version are provided, a warning is logged.
+	 */
+	private _getUrl(): URL {
+		const { definitionsUrl, version } = this.editor.config.get( 'emoji' )!;
+
+		if ( !definitionsUrl || definitionsUrl === 'cdn' ) {
+			// Url was not provided or is set to 'cdn', so we use the default CDN URL.
```

Comment:
To keep the naming consistent.

```suggestion
			// URL was not provided or is set to 'cdn', so we use the default CDN URL.
```